webauthn/registration/generate_registration_options.py

```diff
@@ -126,4 +135,10 @@ def generate_registration_options(
             authenticator_selection.require_resident_key = True
         options.authenticator_selection = authenticator_selection
 
+    options.extensions = AuthenticationExtensionClientInputs(
+        large_blob=large_blob_extension,
+        credential_protection_policy=credential_protection_policy,
+        enforce_credential_protection_policy=enforce_credential_protection_policy,
+    )
```

Comment:
Can you write some tests for this new logic please to ensure the extensions are set when specified? And I'm curious, what would `extensions` out of this method serialize to JSON as if none of the arguments are provided? An empty `{}`? Would they be omitted? I'd like to see `extensions` not get serialized at all if no extensions are specified.